### DISCUSSION TITLE
Counterfactual predictions

### DISCUSSION THREAD

**Discussion by kartechbabu** [DoubleML/doubleml-for-py#240] [Q&A] [answered]
Hello,
Is there any way I can make counterfactual predictions with CI after estimating the causal parameters? If so, what method is available to make these predictions?

Thanks,
Karthik

**Comment by SvenKlaassen** [comment, marked as answer]
Hello,

currently there is no method to make counterfactual predictions (especially with CI since this would include confidence intervals for all the nuisance elements e.g. in the PLR model this would require to also quantify the statistical uncertainty of `g(X)`) implemented.
Generally I would guess that these predictions should be very noisy in most settings.

I guess the closest part would be something like [Heterogeneous treatment effects](https://docs.doubleml.org/stable/guide/heterogeneity.html).

**Reply by kartechbabu** [reply]
Thank you so much, @SvenKlaassen, for your prompt response!
